packages/api/bin/serve.js

```diff
@@ -118,6 +120,14 @@ function checkEnvVariablesAreSet(moreRequiredEnvVars) {
 }
 
 async function createDBRecords(stackName, user) {
+  const esClient = await Search.es('fakehost');
```

Comment:
The local ES instance is initially bootstrapped in `prepareServices`. I think in that function we should set `fakehost` to `process.env.esHost` and `${stackName}-es` to `process.env.esIndex`, then in this function and throughout the endpoint code, we can just use those environment variables? That would also using this method against real stacks more flexible, because the value `localrun-es` is only valid for running API against localstack

Reply:
Definitely yes for passing esIndex, that's just a string. Can we pass esClient? the whole ElasticSearch client? I think I tried this already and it didn't work but I'll test it.

Reply:
No, i wouldn't try setting anything other than string/number to an environment variable. I was just thinking in `prepareServices` we would have

```
process.env.esIndex = `${stackName}-es`;
process.env.esHost = 'fakehost';
```